src/container_of.rs

```diff
@@ -97,4 +97,19 @@ mod tests {
             assert_eq!(container_of!(&x.1, Tup, 1), &x as *const _);
         }
     }
+
+    #[test]
+    fn non_copy() {
+        use core::cell::Cell;
+
+        #[repr(C)]
+        struct Foo {
+            a: Cell<u8>,
```

Comment:
Given the discussions about making `Cell` `Copy`, could you use some more intrinsically non-`Copy` type, like `Box`?

Reply:
I changed it to `RefCell`, which can't be `Copy` by design. Using `Box` is tricky since the crate is `no_std`.